src/firebase.js

```diff
@@ -0,0 +1,27 @@
+import firebase from 'firebase'
+
+const firebaseConfig ={
+    apiKey: "AIzaSyAANq-xa2F5cE2A1jlY7TUWkw3z9v_OE8Y",
```

Comment:
try to pick api key from environment file. Don't expose it like that.

Reply:
it is executing on local side but  i am getting error while deploying  it